test/fixtures/cookbooks/test/recipes/instance.rb

```diff
@@ -19,7 +19,7 @@
   action [:start, :enable]
 end
 
-memcached_instance_sysv_init 'painful_cache' do
+memcached_instance 'painful_cache' do
```

Comment:
The point of this one was to show the non-default provider working. If we're not going to support that use case at this point we should just remove the painful part. Painful was sysv init.